jbpm-flow/src/test/java/org/jbpm/event/process/ProcessEventSupportTest.java

```diff
@@ -543,4 +542,88 @@ public void afterVariableChanged(ProcessVariableChangedEvent event) {
         assertEquals( "org.drools.core.process.event", ((ProcessStartedEvent) processEventList.get(15)).getProcessInstance().getProcessId());
     }
 
+	@Test
+    public void testDefaultParentProcessIdValueInProcessEventListener() throws Exception {
+        InternalKnowledgeBase kbase = KnowledgeBaseFactory.newKnowledgeBase();
+
+        // create a simple package with one process to test the events
+        RuleFlowProcess process = new RuleFlowProcess();
+        process.setId("org.drools.core.process.event");
+        process.setName("Event Process");
+        
+        StartNode startNode = new StartNode();
+        startNode.setName("Start");
+        startNode.setId(1);
+        process.addNode(startNode);
+
+        EndNode endNode = new EndNode();
+        endNode.setName("End");
+        endNode.setId(3);
+        process.addNode(endNode);
+
+        new ConnectionImpl(
+            startNode, Node.CONNECTION_DEFAULT_TYPE,
+            endNode, Node.CONNECTION_DEFAULT_TYPE
+        );
+
+        final InternalKnowledgePackage pkg = new KnowledgePackageImpl( "org.drools.test" );
+        pkg.addProcess(process);
+        List<KiePackage> pkgs = new ArrayList<KiePackage>();
+        pkgs.add( pkg );
+        kbase.addPackages( pkgs );
+
+        KieSession session = kbase.newKieSession();
+        final List<ProcessEvent> processEventList = new ArrayList<ProcessEvent>();
```

Comment:
what's the purpose of this list? I don't see it being checked/asserted in anyway?